deploy/examples/common.yaml

```diff
@@ -548,6 +548,18 @@ rules:
 ---
 kind: ClusterRole
```

Comment:
I would expect the rgw to use a `Role` instead of `ClusterRole`

Reply:
Yaa that's my mistake.